step/serialization.md

```diff
@@ -2,36 +2,43 @@
 
 ## Overview
 
-Serialization has long been a sticky subject in Java. Brian Goetz describing it as a paradox, a key to Java's success allowing for transparent remoting, but also the implementation of serialization making a number of key mistakes. 
+Serialization has long been a sticky subject in Java. Brian Goetz describing it as a paradox, a key to Java's success allowing for transparent remoting, but also the implementation of serialization having a number of critical issues. 
 
-A short list of mistakes include;
-* Bypasses constructors and encapsulation  
+A short list of issues include;
+
+* Breaks encapsulation by ignoring constructors and supplying values to fields via reflection 
 * The behavior of serialization depends on "magic" methods (including `readObject()` which we will look at later)
 * Poor stream format that is not efficent, reusable, nor human readable 
 
-In this section we will explore some of the short comings with Serialization in Java, how some of these issues have been addressed in recent JEPs, and how Records addresses many of the issues with serialization. 
+In this lab we will explore in-practice some of the short comings with serialization in Java, how some of these issues have been addressed in recent JEPs, and how Records addresses many of the remaining issues with serialization. 
+
+## What is Serialization and Deserialization?
```

Comment:
Please start with a sentence that introduce Serialization, and then you can continue with  "Serialization has long been a sticky subject in Java"